Minor section 5 edits (#866)
moq-wg/moq-transport: draft-ietf-moq-transport.md

one.

The receiver of a SUBSCRIBE_ANNOUNCES_OK or SUBSCRIBE_ANNOUNCES_ERROR ought to

> forward the result to the application, so that the application can make decisions
> about further publishers to contact.

Reviewer: ```suggestion
forward the result to the application, so the application can decide which other
publishers to contact.
```

If a publisher is authoritative for a given namespace, or is a relay that has
received an authorized ANNOUNCE for that namespace from an upstream publisher,
it MUST send an ANNOUNCE to any subscriber that has subscribed to ANNOUNCE for

> that namespace, or a superset of that namespace. A publisher MAY send the ANNOUNCE

Reviewer: I thought we decided that you only get namespaces directly under the one you SUBSCRIBE_ANNOUNCE to, and it's not recursive?  Or am I misremembering?

Also, this paragraph is one giant sentence that's difficult to read.

Author: I think SUBSCRIBE/FETCH are routed by direct match only, but ANNOUNCES are not.

I don't know how to fix the sentence.
